Utils/azure-explorer-common/src/com/microsoft/tooling/msservices/helpers/azure/sdk/AzureSDKManager.java

```diff
@@ -70,7 +69,7 @@ public static StorageAccount createStorageAccount(String subscriptionId, String
         }
 
         if (enableEncription) {
-            newStorageAccountWithGroup = newStorageAccountWithGroup.withEncryption(new Encryption());
+            newStorageAccountWithGroup = newStorageAccountWithGroup.withEncryption();
```

Comment:
From the API doc: https://azure.github.io/azure-libraries-for-java/com/microsoft/azure/management/storage/StorageAccount.DefinitionStages.WithEncryption.html

> withEncryption()
Deprecated. 
use StorageAccount.DefinitionStages.WithEncryption.withBlobEncryption() instead.